models/workitemtype.go

```diff
@@ -107,3 +111,22 @@ func (wiType WorkItemType) ConvertFromModel(workItem WorkItem) (*app.WorkItem, e
 
 	return &result, nil
 }
+
+// IsTypeOrSubtypeOf returns true if the work item type is of the given type name,
+// or a subtype; otherwise false is returned.
+func (wit WorkItemType) IsTypeOrSubtypeOf(typeName string) bool {
+	// Remove any prefixed "/"
```

Comment:
I think this could simply be expressed like so:

return wit.name == typename || strings.Contains(wit.Path, "/"+typeName+"/")